### DISCUSSION TITLE
Album decorators : what should I expect as effect?

### DISCUSSION THREAD

**Discussion by agademer** [LycheeOrg/Lychee#2307] [Q&A] [answered]
Hi,

I have some public albums that currently show their thumbnail & name (and that's already cool 😄 ).

I wanted to also show the number of photos that are inside each album.

In the Settings of Lychee I have found the "Album decorations" option
![image](https://github.com/LycheeOrg/Lychee/assets/7581611/9a59f7f4-e395-4ac0-91d1-21e137d44527)

but I don't see any change whatever the option I choose.

Is this option functional ? What should I see in Lychee depending on the option chosen ?

Is it a good way to show the Number of photos (as the name of the option suggest) ? If not, have you an idea of where I should change something to make it happen ?

Thanks in advance.

Regards,

Antoine

**Comment by ildyria** [comment]
Hi,

It was available on v4, I haven't had time to add the functionality on v5 (yes I haven't had time to migrate all the tiny nitty gritty details).

Changes are to be made here:
https://github.com/LycheeOrg/Lychee/blob/master/resources/views/components/gallery/album/thumbs/album.blade.php#L85
and most likely here:
https://github.com/LycheeOrg/Lychee/blob/master/app/View/Components/Gallery/Album/Thumbs/Album.php

with inspiration from:
https://github.com/LycheeOrg/Lychee-front/blob/master/scripts/main/build.js#L192 (from v4)

**Comment by ildyria** [comment, marked as answer]
They are back in v6.
